scaffolder-templates/build/.gitlab-ci.yml

```diff
@@ -0,0 +1,33 @@
+stages:
+  - wait
+  - update
+
+wait_for_value:
+  tags:
+    - shared-podman
```

Comment:
this one is only related to our GL instance no?
Should it be removed or made configurable?